lib/systems/src/PedalsSystem.cpp

```diff
@@ -24,16 +24,17 @@ PedalsSystemData_s PedalsSystem::evaluate_pedals(const AnalogConversion_s &accel
                                                  unsigned long curr_time)
 {
     PedalsSystemData_s out;
-
-    out.accelPercent = ((accel1.conversion + accel2.conversion) / 2.0);
-    out.accelPercent = remove_deadzone_(out.accelPercent, accelParams_.deadzone_margin);
-    out.accelPercent = std::max(out.accelPercent, 0.0f);
+    
+    
     out.accelPressed = pedal_is_active_(accel1.conversion, accel2.conversion, accelParams_, false);
     out.accelImplausible = evaluate_pedal_implausibilities_(accel1, accel2, accelParams_, 0.1);
+    out.accelPercent = (out.accelImplausible) ? accel1.conversion : (accel1.conversion + accel2.conversion) / 2.0;
+    out.accelPercent = remove_deadzone_(out.accelPercent, accelParams_.deadzone_margin);
+    out.accelPercent = std::max(out.accelPercent, 0.0f);
```

Comment:
This doesn't hurt anything, but pedal percent values are already clamped from 0-1